ipc-queue/src/fifo.rs

```diff
@@ -121,6 +121,13 @@ impl<T: Transmittable> Fifo<T> {
             // check pointers are outside enclave range, etc.
             let data = User::<[WithId<T>]>::from_raw_parts(descriptor.data as _, descriptor.len);
             mem::forget(data);
+
+            #[repr(transparent)]
+            #[derive(Copy, Clone)]
+            struct WrapUsize(usize);
+            unsafe impl UserSafeSized for WrapUsize{}
```

Comment:
Move this higher up in the function so that the `UserRef::from_ptr` line is closer to the “check pointers” comment